src/OrchardCore/OrchardCore.Abstractions/Shell/Extensions/ShellFeaturesManagerExtensions.cs

```diff
@@ -34,6 +34,35 @@ public static async Task EnableFeaturesAsync(this IShellFeaturesManager shellFea
         await shellFeaturesManager.EnableFeaturesAsync(featuresToEnable, force: false);
     }
 
+    public static async Task UpdateFeaturesAsync(this IShellFeaturesManager shellFeaturesManager, IEnumerable<string> featureIdsToDisable, IEnumerable<string> featureIdsToEnable)
```

Comment:
Why is this necessary? It's not used.

Reply:
I needed it at some point during this PR. It's probably not a bad idea to keep it as a helpful extension